src/server/helpers/collections.js

```diff
@@ -121,7 +121,7 @@ export async function getCollectionItemBBIDs(collectionId, from, size, orm) {
 						SELECT bookbrainz.user_collection_item.bbid
 						FROM bookbrainz.user_collection_item
 						WHERE collection_id='${collectionId}'
-						ORDER BY user_collection_item.added_at ASC
+						ORDER BY user_collection_item.added_at DESC
```

Comment:
This threw me off at first.
Did you change this so that it's clearer that an element was added to the collection, since it will appear at the top?

I'm not sure how we should display this, but ascending order made more sense to me.
Eventually, we'll want to give the user the freedom to sort however they want with some fancy table library, but for now…